### DISCUSSION TITLE
What is this UI style called?

### DISCUSSION THREAD

**Discussion by ayvacs** [WerWolv/ImHex#1695] [General]
Hey! This isn't a question specifically aimed at ImHex, but I've noticed a ton of programs use this specific UI style (just the general design of the popups, the blue/gray colour scheme, and the font)

I've been wondering for a while if this style is just a coincidence or if it's the result of using a certain UI library or something similar?

Thanks!

![image](https://github.com/WerWolv/ImHex/assets/74518502/0edf0f2e-1d9a-46f9-88f9-6e76623cf78f)

Here's an image of another program that uses the same style for popups:

![image](https://github.com/WerWolv/ImHex/assets/74518502/b561d687-633a-4203-aa2a-0568df3d20e6)

**Comment by glpnk** [comment]
This is [Dear ImGui](https://github.com/ocornut/imgui). Check [this list](https://github.com/ocornut/imgui/wiki/Software-using-dear-imgui) for other projects using it.
